src/lib/util.js

```diff
@@ -35,6 +36,78 @@ function requireFromWorkingDirectory(filename) {
 }
 
 
+/**
+ * Bundle the JS code that creates the reffy namespace in a browser context
+ * out of the source code in src/browserlib.
+ *
+ * @function
+ * @public
+ */
+async function buildBrowserlib() {
```

Comment:
thinking some more about this - is there any reason to use rollup at this point? can't we just insert reffy.js as a module and let the browser do the imports?

Reply:
The same reason as initially ;) I've been using rollup because letting the browser do the imports is not that easy in practice: we want to inject a local script, but imports are going to be resolved against the base URL of the specification. For instance, if you're processing `https://www.w3.org/TR/presentation-api/`, then injecting `reffy.mjs` will trigger requests on `https://www.w3.org/TR/presentation-api/get-title.mjs` because `reffy.mjs` starts with:

```js
import getTitle from './get-title.mjs';
```

Now, the thing is we already intercept network requests, so we can intercept requests on script files and respond with the content of the local files instead. It's a bit convoluted (and cannot be very clean because we cannot use a different domain either, otherwise we'll run into CORS issues), but that seems to work. I'll push something. I'm glad I called the PR branch `shaving-yaks`.